cnames_active.js

```diff
@@ -38,6 +38,7 @@
  */
 
 var cnames_active = {
+  "starrysystems": "starry-systems.github.io",
```

Comment:
```suggestion
  "starrysystems": "starry-systems.github.io/website",
```
If you're wanting to target https://github.com/Starry-Systems/website, then you need to include that path here

Reply:
Also, as the CI (and the file itself) notes, your entry needs to be added in alphabetical order, please.

Reply:
Also also, https://github.com/Starry-Systems/website is not content that we're going to accept in its current state -- it is very clearly just placeholder content: https://github.com/Starry-Systems/website/blob/7f14547c10ff45e76d7029fee429ef7816d416aa/index.html

Reply:
Yeah

Reply:
Ok.